modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSAndesContextStoreImpl.java

```diff
@@ -831,7 +831,55 @@ public void deleteBindingInformation(String exchangeName, String boundQueueName)
             close(connection, RDBMSConstants.TASK_DELETING_BINDING);
         }
     }
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void updateExclusiveConsumerForQueue(String queueName, boolean isExclusiveConsumer) throws AndesException {
+
+        Connection connection = null;
+        PreparedStatement preparedStatement1 = null;
```

Comment:
here also we can add meaningful variable names rather than preparedStatement1 and preparedStatement2